client/src/views/Feed/Feed.vue

```diff
@@ -72,8 +76,31 @@ export default {
       }
     }
   },
+  showNotif () {
+    this.$q.notify({
+      message: 'Oooops, something went wrong',
+      icon: 'announcement'
+    })
+  },
   computed: {
     ...mapGetters('posts', ['getPosts'])
+  },
+  created () {
+    let route = ''
+    if (Authorized.isAuthorized()) {
+      route = '/api/feed/authorized'
+    } else {
+      route = '/api/feed/anonimus'
```

Comment:
For such easy conditions you could use ternary operator to assign result to variable. - https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Reply:
Yes I could, but is it necessary? I think that this way is more readable

Reply:
Ternary more readable for simple conditional statements.